worldeditadditions/utils/parse/init.lua

```diff
@@ -5,11 +5,16 @@
 -- ██      ██   ██ ██   ██ ███████ ███████
 
 local axes = dofile(worldeditadditions.modpath.."/utils/parse/axes.lua")
+local parse = dofile(worldeditadditions.modpath.."/utils/parse/axes.lua")
+local key_instance = dofile(worldeditadditions.modpath.."/utils/parse/key_instance.lua")
 
-worldeditadditions.parse = {
-	axes = axes.parse_axes,
-	axis_name = axes.parse_axis_name
-}
+worldeditadditions.key_instance = key_instance
```

Comment:
If key_instance isn't a child of `parse`, then it shouldn't be defined in the `parse` directory.

Reply:
You've got a point there... where would you suggest I put it instead (given that it's not really useful for anything other than use in this and possibly other parsers)?

Reply:
It should be a local function that is imported via `dofile` in your axis parser, rather than being defined mod-wide.

Reply:
It already is... I guess I don't need to make it globally available.